bot/seasons/christmas/hanukkah_embed.py

```diff
@@ -0,0 +1,113 @@
+import datetime
+import logging
+
+import requests
```

Comment:
`requests` is blocking. You should use `aiohttp` instead, which is already installed as a dependency of d.py.

The [`aiohttp` quickstart](https://aiohttp.readthedocs.io/en/stable/client_quickstart.html) has an example of the equivalent request syntax.